post-processor/shell-local/post-processor.go

```diff
@@ -24,7 +25,7 @@ type Config struct {
 	Inline []string
 
 	// The shebang value used when running inline scripts.
-	InlineShebang string `mapstructure:"inline_shebang"`
+	InlineShebang []string `mapstructure:"inline_shebang"`
```

Comment:
I don't think we can overload this term to mean "shell that executes our script". the `shebang` is really only relevant to inline scripts -- it tells the shell which interpreter to use when evaluating the script.

Since we're not hardcoding `/bin/sh` any more, we probably don't need this. I would remove entirely. It can be bc-breaking if this makes it to 1.1.0 or release in 1.2.0

Maybe we can turn `command` into an array, so `{{.Script}}` becomes `['/bin/bash', '-e', '{{.Script}}']`. That I think would be pretty slick. 

We should let the user of inline scripts write their own shebang into the `inline` field. 

I honestly don't see how to do this in a BC way -- that's okay if it's impossible, but want to hear your thoughts